export const FETCH_ERROR = {
 message: 'Something went wrong, try again later' 
}